server/src/main/java/org/elasticsearch/script/ScoreScript.java

```diff
@@ -84,6 +85,11 @@ public ScoreScript(Map<String, Object> params, SearchLookup lookup, LeafReaderCo
         return params;
     }
 
+    /** The leaf lookup for the Lucene segment this script was created for. */
+    protected final LeafSearchLookup getLeafLookup() {
```

Comment:
Is this accessed from somewhere? I don't see any uses in this PR.

Reply:
@rjernst  Thanks Ryan, I was copying this from a former `SearchScript`, this is removed now